Gemfile.lock

```diff
@@ -403,6 +403,8 @@ GEM
     sqlite3 (1.3.11)
     sse-rails-engine (1.4.0)
       rails (~> 4.0)
+    sucker_punch (2.0.1)
```

Comment:
we have a restart protection in the initializers ... they should wait for sucker-push to be done too instead of killing them ?